doc/fd.1

```diff
@@ -0,0 +1,106 @@
+.TH FD 1 fd-4.0.0
+.SH NAME
+fd \- find files
+.BR find (1)
```

Comment:
What is this for? is it a link to `find`? Do we need this here?

Reply:
Yes, this is a link to find since it's on the current description.